tests/test_json_schema_test_suits_draft4.py

```diff
@@ -0,0 +1,21 @@
+from pathlib import Path
+import pytest
+from test_json_schema_test_suits import template_test, resolve_param_values_and_ids
+
+
+def pytest_generate_tests(metafunc):
+    suite_dir = 'JSON-Schema-Test-Suite/tests/draft4'
+    ignored_suite_files = [
+        'ecmascript-regex.json',
+    ]
+    ignore_tests = []
+
+    suite_dir_path = Path(suite_dir).resolve()
+    test_file_paths = sorted(set(suite_dir_path.glob("**/*.json")))
+
+    param_values, param_ids = resolve_param_values_and_ids(
+        test_file_paths, ignored_suite_files, ignore_tests
+    )
```

Comment:
I think we could pass also dir directly, so it doesn't have to be copy-pasted many times.

Reply:
True... always better to have less code. I will change that